pontoon/base/models.py

```diff
@@ -2360,32 +2364,30 @@ def get_stats(self, locale):
             t for t in translations
             if (t.approved or t.fuzzy) and t.warnings.exists()
         ])
-        unrevieved_count = len([
+        unreviewed_count = len([
             t for t in translations
             if not (t.approved or t.fuzzy or t.rejected)
         ])
 
         if self.string_plural:
             approved = int(approved_strings_count == locale.nplurals)
             fuzzy = int(fuzzy_strings_count == locale.nplurals)
+            if not (approved or fuzzy):
+                if errors_count:
+                    errors_count = 1
+                elif warnings_count:
+                    warnings_count = 1
```

Comment:
Actually you should revert the old code here. We don't need the actual count of errors and warnings. If the translation is neither approved nor fuzzy and contains an error/warning, we count it as such (regardless of plural forms).